index.html

```diff
@@ -1 +1,36 @@
-初期
\ No newline at end of file
+<!DOCTYPE html>
+<html lang="ja">
+
+<head>
+  <meta charset="utf-8">
+  <title>JavaScript Test</title>
+</head>
+
+<body>
+  <h1>ToDoリスト</h1>
+  <input type="radio" name="example" value="選択肢1" checked>すべて
+  <input type="radio" name="example" value="選択肢2">作業中
+  <input type="radio" name="example" value="選択肢3">完了
+  <div id="taskLists">
+    <table id="taskTable">
```

Comment:
idやclassの単語の繋ぎなのですが、HTMLではハイフンで単語を繋ぐ慣習があります。
今回ですと、以下のようになります。
```suggestion
    <table id="task-table">
```
他の箇所も修正してみましょう！